doc/website/concepts/qos-policies.md

```diff
@@ -0,0 +1,33 @@
+# QoS policies
+
+Unlike other middlewares, e.g. [DDS used in ROS 2](https://docs.ros.org/en/galactic/Concepts/About-Quality-of-Service-Settings.html),
+iceoryx keeps things simple and offers basic Quality of Service (QoS) settings. The QoS
+settings are called options and can be used to optimize and tailor the communication.
+
+## `PublisherOptions` and `SubscriberOptions`
+
+The three most important settings are:
+
+| Option                              | Explanation                                                                                                                                       |
+|-------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------|
+| `PublisherOptions::historyCapacity` | This enables late-joining subscribers to request the n last samples                                                                               |
+| `SubscriberOptions::queueCapacity`  | Size of the subscriber queue where samples are stored before they are passed to the user                                                          |
+| `SubscriberOptions::historyRequest` | The number of samples a late-joining subscriber will request from a publisher, should be equal or smaller than `historyCapacity` otherwise capped |
+
+For more information about the options see the corresponding example [`iceoptions`](../getting-started/examples/iceoptions.md).
+
+## Publisher subscriber matching criteria
+
+Two criteria have to be fulfilled in order for a publisher and a subscriber to be connected.
+
+ 1. Same `capro::ServiceDescription`
```

Comment:
Question. Is it possible to have two publishers with the same service description and different policies? So that we have in the end two identical named services to which subscribers can subscribe?

If so then our service handling is broken as well since the QoS is then definitely a part of the service description.

Reply:
This should be possible since those options/QoS do not represent the **structure of the data** but how it is made available (i.e. reliable, how much is cached, will the publisher block etc.). The Servicedescription should from my point of view only be concerned with the former. If we would like to have a DDS analogy it is the type information (or more precisely only a unique identifier for this information).

Note that I dislike the naming and design as it is. I would like to have a **ServiceIdentifier** which would be basically the current **ServiceDescription** which allows me to uniquely identify/locate the corresponding **Service/Type information**. 

The world would then consist of many available topics (services but this terminology is at odds with Autosar) with unique identifiers (say our 3 strings) and there may be many publishers for each of them with **different options**. Finally there are a number of subscribers with their own options which want to get data from a specific topic and will do so if topic and its own requirements *match* in some with what is offered in a way still to be precisely defined. This matching behavior hence depends on the **ServiceIdentifier AND the options** and is what this document seems to be about...

Reply:
@elfenpiff Yes, this is currently possible. The QoS options are independent from the `ServiceDescription`. We won't change this before `v2.0`, hence the description presents the current status quo.